### ISSUE TITLE
es modules support undone in 2.2.1

### ISSUE THREAD

**Issue by leoliu** [andreknieriem/simplelightbox#176]
Check `dist/simple-lightbox.js` to see there is no `exports`.

**Comment by andreknieriem** [comment]
Hi, you need to use the https://github.com/andreknieriem/simplelightbox/blob/master/dist/simple-lightbox.modules.js version. If I put in the main version and use it without webpack or any of these tools, I get an exports is not defined and the plugin crashes. So I moved it to an own file.

**Comment by ElectroluxV2** [comment]
Maybe create separate npm module instead? It's a bit ugly to import things like that.

**Comment by csalmeida** [comment]
I have the same issue when importing it to the project:

```bash
Error: 'default' is not exported by node_modules/simplelightbox/dist/simple-lightbox.modules.js
```

Tried importing the following ways, the first one mentioned in the documentation. The version I'm using is [`v2.2.1`](https://github.com/andreknieriem/simplelightbox/releases/tag/v2.2.1):

1.

```javascript
import SimpleLightbox from "simplelightbox";
```

2.

```javascript
import SimpleLightbox from "simplelightbox/dist/simple-lightbox.modules";
```

Using `import { SimpleLightbox } from "simplelightbox";` produces a similar error, pointing out there are no exports:

```bash
Error: 'SimpleLightbox' is not exported by node_modules/simplelightbox/dist/simple-lightbox.js
```

I've noticed that in [`src/modules.js` ](https://github.com/andreknieriem/simplelightbox/blob/master/src/modules.js)there is an export but it seems to not be working at all, could this be a bug? 

Any advice on how to import the project using a bundle at this stage would be greatly appreciated! 🙏
